### DISCUSSION TITLE
How to get the app url?

### DISCUSSION THREAD

**Discussion by myckhel** [adonisjs/core#3054] [Help] [answered]
Please someone rescue me.
i have searched the entire world of internet and could not find anything useful.

How do i get the app url in adonisJs 5 so that it gives me something like http://127.0.0.1:3333.

Like in `laravel` you can get the app url in different ways.
```php
// Controller.php

route('users.index');

config('app.url');
```

**Comment by thetutlage** [comment, marked as answer]
What is value of `url` property inside `config/app.php` file in Laravel?

**Reply by myckhel** [reply]
https://github.com/laravel/laravel/blob/b55e3fbb14ab8806057f451fcbfeec8fca625793/config/app.php#L55

Here is the referenced content

**Reply by thetutlage** [reply]
As you can notice, it is using the environment variable. You can do same with AdonisJS as well

**Reply by myckhel** [reply]
Thank you.

I will do that.
